src/Microsoft.DotNet.Interactive.ExtensionLab/MsSqlServiceClient.cs

```diff
@@ -156,6 +167,24 @@ public async Task<QueryExecuteSubsetResult> ExecuteQueryExecuteSubsetAsync(strin
             return await rpc.InvokeWithParameterObjectAsync<QueryExecuteSubsetResult>("query/subset", queryExecuteSubsetParams);
         }
 
+        public async Task SendTextChangeNotification(string ownerUri, string newText, string oldText)
+        {
+            var oldLines = oldText.Split('\n');
```

Comment:
This should probably also anticipate `\r\n`.

Reply:
Oh, I see you're just using this to count the lines. That can be made a little more explicit given the use case by doing something like this:

```csharp
var numberOfLines = oldText.Split(`\n`).Length;
```